hi-ml-azure/src/health_azure/himl.py

```diff
@@ -560,12 +564,18 @@ def submit_run_v2(workspace: Optional[Workspace],
             shm_size=docker_shm_size,
             display_name=display_name,
             instance_count=num_nodes,
-            distribution={
-                "type": "PyTorch",
-                "process_count_per_instance": processes_per_node,
-            },
+            distribution=distribution,  # type: ignore
         )
 
+    if hyperparam_args:
+        param_sampling = hyperparam_args[PARAM_SAMPLING_ARG]
+
+        for sample_param, choices in param_sampling.items():
+            input_datasets_v2[sample_param] = choices.values[0]
+            cmd += f" --{sample_param}=" + "${{inputs." + sample_param + "}}"
```

Comment:
Do we want to manually add this to the command? I thought AML did already this for each of the children